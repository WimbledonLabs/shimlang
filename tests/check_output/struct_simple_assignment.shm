struct Foo {
    a,
    b,
    c,

    fn print(self) {
        print("a =", self.a);
        print("b =", self.b);
        print("c =", self.c);
    }
}

let x = Foo {
    c=0;
    b=0;
    a=0;
};

x.a = 42;
x.b = 41;
x.c = 40;

x.print();
