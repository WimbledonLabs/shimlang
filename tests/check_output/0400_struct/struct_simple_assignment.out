a = 42
b = 41
c = 40
